.circleci/config.yml

```diff
@@ -66,23 +78,72 @@ jobs:
           key: v2-node-{{ arch }}-{{ checksum "yarn.lock" }}-yarn
           paths:
             - ~/.cache/yarn
+      # Store node_modules for all jobs in this workflow so that they don't
+      # need to each run a yarn install for each job. This will speed up all
+      # jobs run on this branch with the same lockfile. Split into multiple
+      # caches as there's a limited number of paths per cache.
```

Comment:
This is the specific error we get if we put everything in one cache: https://support.circleci.com/hc/en-us/articles/360037695934-save-cache-step-fails-with-Error-uploading-archive-MetadataTooLarge-Your-metadata-headers-exceed-the-maximum-allowed-metadata-size-